profiles/common/modules/features/nexteuropa_multilingual/nexteuropa_multilingual.install

```diff
@@ -39,6 +39,10 @@ function nexteuropa_multilingual_install() {
   // Set translatable states for TMGMT Workbench module.
   multisite_config_service('system')->setVariable('tmgmt_workbench_moderation_states', array('validated', 'published'));
 
+  // Disable multisite_drupal_language_negociation (redundant with
+  // nexteuropa_multilingual).
+  module_disable(array('multisite_drupal_language_negociation'));
```

Comment:
This line was added 6 months ago on the platfrome, and I deleted it by error on the previous commit.This code revert the mistake.